src/DotVVM.Samples.Tests/Feature/ApiTests.cs

```diff
@@ -202,7 +202,7 @@ public void Feature_Api_AzureFunctionsApiTable()
             });
         }
 
-        [Fact]
+        [Fact(Skip = "Doesn't work on OWIN because it relies on _apiCore.")]
```

Comment:
In the future it would be nice to have SkipOnOwin / SkipOnAspNetCore attributes.

Reply:
Agreed. Please create a separate issue.